packages/core/src/tests/constants.ts

```diff
@@ -141,3 +141,134 @@ export enum NavigationCommand {
   PAGE_UP = "{pageup}",
   PAGE_DOWN = "{pagedown}"
 }
+
+export enum ComponentVibeId {
+  TRANSITION_VIEW = "TransitionView",
+  TEXT_AREA = "TextArea",
+  EDITABLE_TEXT = "EditableText",
+  TIPSEEN_MEDIA = "TipseenMedia",
+  INDICATOR = "Indicator",
+  BADGE = "Badge",
+  TITLE = "Title",
+  TEXT = "Text",
+  COMBOBOX = "Combobox",
+  COMBOBOX_CATEGORY = "ComboboxCategory",
+  COMBOBOX_OPTION = "ComboboxOption",
```

Comment:
These can be removed, I think only Combobox is relevant, the subcomponent don't matter